testdata/symlinks/public

```diff
@@ -0,0 +1 @@
+test
```

Comment:
```suggestion
test

```

Reply:
This is a symlink, not a text file. I don't think we should be changing the content